packages/web-backend/src/Videos/index.ts

```
@@ -0,0 +1,111 @@
+import { Array, Effect, Option, Schema, Stream } from "effect";
```

Biome (2.1.2)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)